mkFit/deadmodules.h

```diff
@@ -0,0 +1,696 @@
+deadvectors[70].push_back({1.79387,1.90873,72.787,91.2269});
+deadvectors[70].push_back({-2.8063,-2.69143,72.7778,91.2158});
+deadvectors[70].push_back({2.39452,2.47495,88.9094,109.398});
+deadvectors[70].push_back({-1.53249,-1.45203,88.8906,109.379});
+deadvectors[27].push_back({-1.34772,-1.23288,72.7945,91.2337});
+deadvectors[28].push_back({-0.639305,-0.458593,60.015,75.7957});
+deadvectors[27].push_back({-0.629566,-0.469945,60.5178,75.2913});
+deadvectors[28].push_back({-0.482252,-0.301548,60.0159,75.7963});
+deadvectors[27].push_back({-0.472505,-0.312887,60.5182,75.2915});
+deadvectors[28].push_back({-0.325148,-0.14444,60.0161,75.7967});
+deadvectors[27].push_back({-0.315396,-0.155779,60.5201,75.2936});
+deadvectors[27].push_back({-0.590001,-0.509565,88.9007,109.389});
+deadvectors[27].push_back({-0.511437,-0.431001,88.9011,109.389});
+deadvectors[27].push_back({-0.432919,-0.352485,88.9027,109.391});
+deadvectors[27].push_back({-0.354367,-0.273934,88.9036,109.392});
+deadvectors[27].push_back({-0.275852,-0.195412,88.8985,109.387});
+deadvectors[27].push_back({-0.26395,0.0021703,23.7177,32.3097});
+deadvectors[28].push_back({1.55748,1.84618,32.122,41.949});
+deadvectors[28].push_back({1.40273,1.58343,60.0179,75.7985});
+deadvectors[28].push_back({3.1306,-2.97188,60.0197,75.8003});
+deadvectors[29].push_back({0.509557,0.590004,88.8957,109.384});
+deadvectors[29].push_back({1.29495,1.3754,88.8918,109.38});
+deadvectors[29].push_back({-1.12327,-1.00839,50.5036,62.039});
+deadvectors[30].push_back({-2.63137,-2.34265,32.1209,41.9484});
+deadvectors[29].push_back({-1.45991,-1.34507,72.7944,91.2336});
+deadvectors[30].push_back({-1.06379,-0.768282,23.371,32.6555});
+deadvectors[30].push_back({-0.953477,-0.772774,60.0148,75.795});
+deadvectors[31].push_back({2.01827,2.13313,50.5025,62.0375});
+deadvectors[32].push_back({1.87395,2.05465,60.0149,75.7949});
+deadvectors[32].push_back({-2.89638,-2.6009,23.3724,32.6567});
+deadvectors[32].push_back({-2.11099,-1.8155,23.3728,32.6575});
+deadvectors[31].push_back({0.0767547,0.237525,39.3377,50.4399});
+deadvectors[31].push_back({0.67191,0.786754,72.7941,91.2335});
+deadvectors[31].push_back({0.58809,0.668528,88.8989,109.387});
+deadvectors[31].push_back({1.56948,1.68432,72.7971,91.2369});
+deadvectors[31].push_back({1.60911,1.68955,88.9008,109.389});
+deadvectors[32].push_back({2.86642,-3.12804,32.1204,41.9479});
+deadvectors[31].push_back({2.91587,3.03073,50.5025,62.0376});
+deadvectors[31].push_back({-3.10327,-3.02282,88.8888,109.377});
+deadvectors[31].push_back({-2.35752,-2.24266,50.504,62.0391});
+deadvectors[31].push_back({-1.45992,-1.34505,50.5016,62.0366});
+deadvectors[31].push_back({-1.57206,-1.41244,60.5178,75.2913});
+deadvectors[33].push_back({1.29496,1.3754,88.9013,109.39});
+deadvectors[33].push_back({1.83039,2.09648,32.5881,41.4833});
+deadvectors[33].push_back({-1.34772,-1.23286,50.5025,62.0376});
+deadvectors[33].push_back({3.14024,-3.02807,50.5027,62.0379});
```

Comment:
what is that will cause troubles?

Reply:
{3.14024,-3.02807,50.5027,62.0379} ... wrap around pi

Reply:
This actually needs to be fixed before merge. Giuseppe, I'd propose to do it in the generator, then it will be automatically fixed for cmssw as well, right?
Something like:
```
if (phi_low > phi_high) { // Need more sanity checks?
   emit_for(phi_low, Config::PI, ...);
   emit_for(-Config::PI, phi_high, ....);
}
```

Reply:
it's not wrapping around pi=3.14159265359. I am not sure this is an issue

Reply:
`{3.14024,-3.02807,...}` translates to bin numbers ~127 and ~0, respectively. The fill loop ends up with `(i=127; i<0; ++i)` :)

Reply:
I pushed a fix that handles this in LayerOfHits::SuckInDeads(). You decide if you want to keep it like this or correct it in the producer -- probably it depends on how this is handled in CMSSW.

Reply:
thanks! Yes having it here seems safe.